Xamarin.Forms.Platform.Tizen/Shell/ShellRenderer.cs

```diff
@@ -119,6 +121,28 @@ void UpdateFlyoutBackgroundColor()
 			_navigationView.BackgroundColor = Element.FlyoutBackgroundColor.ToNative();
 		}
 
+		void UpdateFlyoutBackgroundImageAspect()
+		{
+			if(_navigationView.BackgroundImage != null)
+			{
+				_navigationView.BackgroundImage.Aspect = Element.FlyoutBackgroundImageAspect;
+			}
+		}
+
+		void UpdateFlyoutBackgroundImage()
+		{
+			if (Element.FlyoutBackgroundImage != null)
+			{
+				_navigationView.BackgroundImage = new Native.Image(Forms.NativeParent);
+				_navigationView.BackgroundImage.Aspect = Element.FlyoutBackgroundImageAspect;
+				var task = _navigationView.BackgroundImage.LoadFromImageSourceAsync(Element.FlyoutBackgroundImage);
```

Comment:
when is this task used?

Reply:
Fixed.  and this codes are moved to NavigationView class.
The return value of LoadFromImageSourceAsync method will be ignored because we don't need to wait until the loading is done.